### PR TITLE
Update travis ruby versions

### PR DESCRIPTION
**Pull request by dnkoutso**
See https://github.com/travis-ci/travis-ci/issues/8302

### CODE REVIEW

**Comment by dnkoutso** [comment]
...or we decide to use another ruby version that works with Trusty from here http://rubies.travis-ci.org/

**Comment by segiddins** [comment]
We test that ruby version specifically since it used to come pre-installed on OS X. Also note that precise is deprecated?

**Comment by dnkoutso** [comment]
Looks like it. But I dont think it will go away...

> By the end of September 2017, we hope to have 100% of incoming configurations with dist: precise routed to the sudo-enabled infrastructure, and to retire the container-based infrastructure. We have no immediate plans to discontinue support for jobs that specify sudo: required and dist: precise, although we do not plan to update the images for these jobs any more.

File: .travis.yml [hunk 1]
```diff
@@ -14,13 +14,13 @@ matrix:
...
-  - 2.0.0-p481
+  # OS X 10.9.5-10.10.0 (2.0.0-p647)
+  - 2.0.0-p647
   # OS X 10.9.3-10.9.4
-  - 2.0.0-p451
+  - 2.0.0-p647
   # OS X 10.9.0-10.9.2
   # TODO: currently unavailable: https://github.com/travis-ci/travis-ci/issues/2918
-  # - 2.0.0-p247
+  # - 2.0.0-p647
```
Comment (segiddins) [inline .travis.yml]: no need to test it twice, and let's remove the outdated comments

**Pull request merged**